security/config.py

```diff
@@ -13,17 +13,111 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import os
+CHECKER_DEFAULT_CONF_PATH = "/etc/oss/security/checker.yaml"
 
-import yaml
+CHECKER_DEFAULT = {
+    "regions": [],
+    "elastic": {
+        "hosts": [],
+    },
+    "plugins": [],
+}
 
+ELASTIC_SCHEMA = {
+    "elastic": {
+        "type": "object",
+        "properties": {
+            "hosts": {
+                "type": "array",
+                "minItems": 1,
+                "uniqueItems": True,
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "host": {
+                            "type": "string",
+                            "minLength": 1,
+                        },
+                        "port": {"type": "integer"},
+                    },
+                    "required": ["host"],
+                    "additionalProperties": False,
+                },
+            },
+        },
+        "required": ["hosts"],
+        "additionalProperties": False,
```

Comment:
User may want to add some extra arguments like described here: https://elasticsearch-py.readthedocs.io/en/master/#ssl-and-authentication